app/src/components/integrations/elements/integrationSettings/integrationSettings.jsx

```diff
@@ -80,7 +80,10 @@ export const IntegrationSettings = (props) => {
   const testIntegrationConnection = useCallback(() => {
     if ('id' in props.data && !props.preventTestConnection) {
       setLoading(true);
-      fetch(URLS.testIntegrationConnection(projectId || activeProject, props.data.id))
+      Promise.all([
+        fetch(URLS.testIntegrationConnection(projectId || activeProject, props.data.id)),
+        fetch(URLS.testIntegrationAvailability(props.data.id)),
+      ])
```

Comment:
I suppose we don't need to test connetection for project&global integration at the same time.
We can deal with one type of integrations on UI at the same time. So I suppose you need to use `isGlobal` to determine which type is used to send the appropriate request.